server/src/server.ts

```diff
@@ -67,7 +67,7 @@ let send: (msg: p.Message) => void = (_) => {};
 
 let getBinaryDirPath = (projectRootPath: p.DocumentUri) =>
```

Comment:
This is the only called. Feels like the two functions are tightly coupled and should not live in different files.
Too easy to change one without adapting the other.

Reply:
In fact why do we need 2 functions to begin with.